tests/functional/spec/choose_another_section.spec.js

```diff
@@ -0,0 +1,17 @@
+const helpers = require('../../../helpers');
```

Comment:
I think this should live as part of the `hub_and_spoke` spec folder since this is currently a hub specific implementation?. Thoughts?